zarr-implementations-council.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: implementations council
+description: Representatives of various Zarr Implementations
+nav_order: 4
+permalink: /zic/
+---
+
+# Zarr Implementation Council 🚀
+
+The [ZSC](https://github.com/zarr-developers/governance/blob/main/GOVERNANCE.md#zarr-steering-council) have invited Zarr Implementations to participate in the management of the Zarr specification through the Zarr Implementation Council (ZIC). Implementations are selected based on the maturity of implementation as well as the activity of the developer community. Preference will be given to open-source and *open-process* implementations. Multiple implementations in a single programming language may be invited, or such implementations could work together as a single community.
+
+The current list of implementations which are participating in this process are (in alphabetical order):
+
+- [constantinpape/z5](https://github.com/constantinpape/z5) represented by [Constantin Pape](https://github.com/constantinpape) ([May 2022 – present](https://github.com/zarr-developers/governance/issues/26))
```

Comment:
Would you suggest moving this info here from governance or duplicate?

Reply:
I'd prefer if we keep it in both places.